packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -221,7 +217,7 @@ export const Snackbar: FC<SnackbarProps> = (toast) => {
 
   const buttonContainer: ViewProps = {
     style: {
-      flex: 0,
+      flexGrow: 0,
```

Comment:
This was changed because it seems otherwise react-native-web (or web flexbox generally) does not respect the flex 0 and `iconAndMessageContainer` just ate all the available space, even when it didn't need it at all. This appeared to result in no functional change in iOS/Android.